src/shapes/ActiveSelection.ts

```diff
@@ -86,7 +86,9 @@ export class ActiveSelection extends Group {
       //  save ref to group for later in order to return to it
       const parent = object.group;
       parent._exitGroup(object);
-      object.__owningGroup = parent;
+      // make sure we are setting the correct owning group
+      // in case `object` is transferred between active selections
```

Comment:
there shouldn't be transferring between active selections.
How that happens?

Reply:
Now you can do:
```
new ActiveSelection(canvas.getActiveObjects())
```
And that will hit this code